lib/recurly.js

```diff
@@ -66,6 +71,7 @@ export class Recurly extends Emitter {
       token: bind(this, bankAccount.token),
       bankInfo: bind(this, bankAccount.bankInfo)
     };
+    this.fraud = new Fraud(this);
```

Comment:
Yeah totally. I think a concept of parent/child context will clarify that considerably.